### DISCUSSION TITLE
23.10.06 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#22] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 트랜잭션 세분화
  - 패키지 정리
- 내일은 어떤 일을 하나요?
  - PR 작성

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
    - 플레이리스트 등록 서비스 테스트 작성
    - 플레이리스트 서비스 비즈니스 로직 구현
- 내일은 어떤 일을 하나요?
    - 플레이르스트 등록 유효성 추가
    - 플레이리스트 조회 서비스 테스트 작성 및 구현

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - JWT 발급 로직 제외
    - spotify에서 내려준 access token 보관에 대한 고민..🤔
- 내일은 어떤 일을 하나요?
    - PR 작성 및 코드리뷰
    - 테스트 코드 작성

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 주문 생성 및 조회 기능 / 테스트 코드 추가
* 내일은 어떤 일을 하나요?
  * 결제를 어디까지 구현해볼지 생각해보기
